src/features/collectables/NftList.tsx

```diff
@@ -39,7 +39,10 @@ const NftList = () => {
       item: string
     }) => {
       return (
-        <NFTListItem item={token} collectables={collectablesWithMeta[token]} />
+        <NFTListItem
+          item={collectablesWithMeta[token][0].symbol}
```

Comment:
is `collectablesWithMeta[token][0].symbol` guaranteed to always exist?

Reply:
It is. At least for the helium nfts